src/components/Cards/MinutesItemCard/MinutesItemCard.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from "react";
+
+import "@mozilla-protocol/core/protocol/css/protocol.css";
+
+interface Docs {
+  /**Document item label */
+  label: string;
+  /**Document item url */
+  url: string;
+}
+
+interface Items {
+  /**Minute item headline */
+  item: string;
+  /**Array of documents to be shown below */
+  docs?: Docs[];
+}
+
+export interface MinutesItemProps {
+  minutesItems: Items[];
+}
+
+function returnDocList(docs: Docs[] | undefined) {
+  if (docs !== undefined) {
+    return (
+      <section>
+        <details>
+          <summary style={{ color: "blue" }}>
+            <p>See documents</p>
+          </summary>
+          <ul className="mzp-u-list-styled" style={{ listStyleType: "none" }}>
```

Comment:
If we're going to remove the bullet points from the list I wonder if it would be nicer to make the indentation level inline with the "See Documents" text?

@tohuynh @JacksonMaxfield Thoughts?